.gitignore

```diff
@@ -28,3 +28,4 @@ packages/hardhat-test/artifacts
 examples/*/yarn.lock
 yarn-error.log
 packages/test-e2e/test/*/out
+.DS_Store
```

Comment:
Please move this to your global git ignore filee

Reply:
done